packages/application/style/code-snippet.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'>
+<svg height="500px" id="Layer_1" version="1.1" viewBox="125 150 250 200" width="500px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g>
+        <g>
+            <polygon fill="#231F20" points="195.568,185.811 142.681,250.173 195.568,314.534 208.077,293.962 172.44,250.173      208.077,206.384    "/>
+            <polygon fill="#231F20" points="228.707,313 251.922,313 288.816,187 265.6,187    "/>
+            <polygon fill="#231F20" points="303.876,185.81 291.37,206.384 327.005,250.173 291.37,293.961 303.876,314.535 356.765,250.173         "/>
```

Comment:
not a big deal, but perhaps remove the extra spaces in the `points`
```suggestion
            <polygon fill="#231F20" points="195.568,185.811 142.681,250.173 195.568,314.534 208.077,293.962 172.44,250.173 208.077,206.384"/>
            <polygon fill="#231F20" points="228.707,313 251.922,313 288.816,187 265.6,187"/>
            <polygon fill="#231F20" points="303.876,185.81 291.37,206.384 327.005,250.173 291.37,293.961 303.876,314.535 356.765,250.173"/>
```

Reply:
I actually just copied the svg file "as is" and then added some newlines for readability before I edited the viewbox. I committed your suggestion though cause I see no reason not to.